### PR TITLE
Confdeck 175

### PR DESCRIPTION
**Pull request by beydogan**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ gem 'omniauth-google-oauth2'
 gem 'omniauth-github'
 gem 'virtus'
 gem "i18n-js", ">= 3.0.0.rc8"
-gem 'bugsnag'
+#gem 'bugsnag'
```
Comment (xaph) [inline Gemfile:43]: dont't comment out bugsnag. move it into production.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -36,14 +36,11 @@ def after_sign_in_path_for(resource)
     end
   end
 
-  private
```
Comment (xaph) [inline app/controllers/application_controller.rb:39]: @beydogan why did you remove this?

**Comment by beydogan** [comment]
@xaph fixed. merging
